pkg/sentry/fs/host/tty.go

```diff
@@ -98,8 +100,11 @@ func (t *TTYFileOperations) Write(ctx context.Context, file *fs.File, src userme
 	defer t.mu.Unlock()
 
 	// Are we allowed to do the write?
-	if err := t.checkChange(ctx, linux.SIGTTOU); err != nil {
-		return 0, err
+	// drivers/tty/n_tty.c:n_tty_write()=>tty_check_change()
```

Comment:
Comments should be written as sentences, so how about:
```
// Check whether TOSTOP is enabled. This corresponds to the check in
// drivers/tty/n_tty.c:n_tty_write().
```

Reply:
Ah, sorry, have breaked this too many times. Your comment sounds great to me. Will do the change.